engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oidc/OidcAuthenticationResources.scala

```diff
@@ -6,7 +6,7 @@ import sttp.client.{NothingT, SttpBackend}
 import scala.concurrent.{ExecutionContext, Future}
 
 
-class OidcAuthenticationResources(realm: String, configuration: OidcAuthenticationConfiguration)(implicit ec: ExecutionContext, sttpBackend: SttpBackend[Future, Nothing, NothingT])
+class OidcAuthenticationResources(override val name: String, realm: String, configuration: OidcAuthenticationConfiguration)(implicit ec: ExecutionContext, sttpBackend: SttpBackend[Future, Nothing, NothingT])
```

Comment:
I wonder, maybe it would be better to just pass it as constructor parameter? Now it's not quite clear for me, looking at the code, when and how this should be overridden

Reply:
Hmmm... I don't get it. `override` does not mean that a value or method is to be overridden, but that it overrides a value or method from a superclass.